app/services/api/v3/dashboards/chart_parameters.rb

```diff
@@ -12,7 +12,10 @@ class ChartParameters
                     :sources_ids,
                     :companies_ids,
                     :destinations_ids,
-                    :nodes_ids_by_position
+                    :node_type,
+                    :nodes_ids_by_position,
+                    :top_n
+        TOP_N = 20
```

Comment:
buried constants = evil?

Reply:
do you think it should go in the controller instead? this is a default in case the top_n parameter was not given in params

Reply:
yeah, I noticed it's a fallback. I think sort of thing is best placed elsewhere - controller would be a good option, as it will be the entry point for us-in-6-months when we want to change it, and also allows the service to be decoupled from "default to 20", which is a more endpoint-related decision